docs/screenplay_elements/API/Questions/response.md

```diff
@@ -104,4 +106,26 @@ public beenReceivedWithin(response: ResponseType, duration: number): Response;
   - `duration` - The expected duration (in milliseconds).
 - **Returns:** `Response` - The updated instance of the `Response` class.
 
+### withAbilityAlias
+
+```typescript
+public withAbilityAlias(alias: string): Response;
+```
+
+- **Description:** Defines the ability alias to be used during execution.
+- **Parameters:**
+  - `alias` - The alias.
+- **Returns:** `Response` - Returns the current question.
+
+### failAsFalse
+
+*Introduced in: 1.6.0*
+
+```typescript
+public get failAsFalse(): Response;
+```
+
+- **Description:** Returns false instead of failing when exception occurrs.
+- **Returns:** `Response` - Returns the current action.
```

Comment:
Oben in Zeile 118 steht: **Returns:** `Response` - Returns the current question.
und hier `the current action`?